mask/eolearn/mask/masking.py

```diff
@@ -45,9 +53,9 @@ def __init__(self, input_features, output_feature, join_operation: Union[str, Ca
 
     def zip_method(self, *masks: np.ndarray) -> np.ndarray:
         """Joins masks using the provided operation"""
-        final_mask, *masks = masks
+        final_mask, *masks = masks  # type: ignore[assignment]
         for mask in masks:
-            final_mask = self.join_method(final_mask, mask)
+            final_mask = self.join_method(final_mask, mask)  # type: ignore[operator]
```

Comment:
you should only ignore if you need to. There is no point in type-checking if we then ignore the types